packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md

```diff
@@ -6,6 +6,26 @@ title: Okta Identity Engine API Products release notes 2023
 
 ## April
 
+### Weekly release 2023.04.2
+
+| Change | Expected in Preview Orgs |
+| ------ | ------------------------ |
+| [Bugs fixed in 2023.04.2](#bugs-fixed-in-2023-04-2) | April 26, 2023 |
+
+#### Bugs fixed in 2023.04.2
+
+- When the sign-in page was edited using the code editor, the event type `system.custom_error.update` was logged. (OKTA-591800)
+
+- The Policy API `constraints` object could be set with the `types` and `methods` of disabled authenticators or methods. (OKTA-586221)
+
+- A "Next" link wasn't returned when a List Identity Providers request was made and a limit of 200 was set. (OKTA-597359)
```

Comment:
```suggestion
- A **Next** link wasn't returned when a List Identity Providers request was made and a limit of 200 was set. (OKTA-597359)
```
Not sure if we're supposed to use quotes?

Reply:
Yeah, I wondered about that. I think I'll use code font: https://developer.okta.com/docs/reference/core-okta-api/#link-header